website/docs/docs/dbt-cloud/access-control/access-control-overview.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Access Control"
+id: "access-control-overview"
+---
+
+## Overview
+
+dbt Cloud administrators can use dbt Cloud's permissioning model to restrict the
+permissions of users on their account. This access control comes in two
+flavors: License-based, and Role-based.
+
+- **License-based Access Controls:** User are configured with account-wide
+  license types. These licenses control the specific parts of the dbt Cloud application
+  that a given user can access.
+- **Role-based Access Control (RBAC):** Users are assigned to _groups_ that have
+  specific permissions on specific projects or the entire account. A user may be
+  a member of multiple groups, and those groups may have permissions on multiple
+  projects.
+
+## License-based access control
+
+Each user on an account is assigned a license type when the user is first
+invited to a given account. This license type may change over time, but a
+user can only have one type of license at any given time.
+
+A user's license type controls the features in dbt Cloud that the user is able
+to access. dbt Cloud's two license types are:
+ - **Read Only**
+ - **Developer**
+
+For more information on these license types, see [Seats & Users](cloud-seats-and-users).
+At a high-level, Developers may be granted _any_ permissions, whereas Read Only
+users will have read-only permissions applied to all dbt Cloud resources
+regardless of the role-based permissions that the user is assigned.
+
+## Role-based access control
+
+Role-based access control allows for fine-grained permissioning in the dbt Cloud
```

Comment:
Should we note that for Self-Service, there is only member and owner?

Reply:
good call - I added a callout to this section